style.css

```diff
@@ -150,7 +150,6 @@ button,
 input,
```

Comment:
Should we update the version at the top of this CSS file for the theme version itself to match functions.php?

From this PR in the functions.php file:

`define( 'MEMBERLITE_VERSION', '4.5.4.1.91' );`

Reply:
The version should probably be `4.5.4` in both here and functions.php

Reply:
> [I] do that right before I merge dev into master.

Sounds good here